src/main/java/com/bench/project/service/TextProcessingService.java

```diff
@@ -34,4 +36,15 @@ public void process(ProcessTextRequest request) {
             }
         );
     }
+
+    public List<LogDto> getResults() {
+        return dao.getAll();
+    }
+
+    public List<LogDto> getResults(String id) {
+        return dao.getAll()
+                .stream()
+                .filter(logDto -> logDto.id().equals(id))
```

Comment:
What if we store results as a map (request id to request object) instead of a list in dao, so we can remove this filtering?